src/apps/Calculators/AxByEqualsP/AxByModeReadoutTable.js

```diff
@@ -0,0 +1,67 @@
+import { handleSetPlayerClick } from "../../../eventHandlers";
+import generateAxByScl from "./util/generateAxByScl";
+
+const AxByModeReadoutTable = ({
+  index,
+  ltObject,
+  sclData,
+  stateData,
+  setStateData,
+}) => {
+  const setPlayerObject = {
+    tool: "axby",
+    scale: ltObject.mode,
+    sclData: sclData,
+    mode: index + 1,
+  };
+  return (
+    <div>
+      <br />
+      <h2>
+        Mode {index + 1} - {ltObject.lmsList.join("")}
+        <br />
+        <button
+          className="mb"
+          onClick={() =>
+            handleSetPlayerClick(stateData, setStateData, setPlayerObject)
+          }
+        >
+          Use In Player
+        </button>
+        <button
+          className="mb"
+          onClick={() => generateAxByScl(ltObject.mode, sclData, index + 1)}
+        >
+          Export .scl
+        </button>
+      </h2>
+      <br />
+      <table>
+        <tbody>
+          <tr>
+            <td className="tableTopRow">Step</td>
+            <td className="tableTopRow">Value</td>
+            <td className="tableTopRow">Difference</td>
+            <td className="tableTopRow">Interval</td>
```

Comment:
Even little things like this can be simplified by looping:

```suggestion
            {["Step", "Value", "Difference", "Interval"].map((name) => (
              <td className="tableTopRow">{name}</td>
            ))}
```